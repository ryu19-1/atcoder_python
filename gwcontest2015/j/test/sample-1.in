6
2 1 2 2
1 1 1000000 1000000
3 2 5 3
5 2 4 3
2015 55 1700 1300
100 50 1000 500
