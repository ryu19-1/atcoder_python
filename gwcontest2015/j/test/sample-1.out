2
1
42
252
926737422
143485143
